### PR TITLE
[Merged by Bors] - Update README.md

### PR DESCRIPTION
**Pull request by morenol**
Bump version in readme and add an example for secrets

### CODE REVIEW

**Comment by EstebanBorai** [comment]
I think CI gets fixed by #35, right? @morenol

File: README.md [hunk 1]
```diff
@@ -58,6 +58,30 @@ The produced record in Fluvio topic will be:
   "length": 158
 }
 ```
+### Secrets
+
+Fluvio HTTP Source Connector support Secrets in the `endpoint` and in the `headers` parameters:
```
Comment (morenol) [inline README.md:63]:
```suggestion
Fluvio HTTP Source Connector supports Secrets in the `endpoint` and in the `headers` parameters:
```

**Review by digikata — APPROVED** [review]
LGTM!

**Pull request closed without merging**